playbooks/ansible-cloud/vcsa-appliance/pre.yaml

```diff
@@ -8,10 +8,52 @@
         name: sshpass
         state: present
 
-    - name: Setup tox role
```

Comment:
This will not work, as zuul doesn't have have SSH info to connect to this VM, this is why we still need embedded bootstrap.yaml files.